### PR TITLE
[zh-cn]sync pull-image-private-registry

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/configure-pod-container/pull-image-private-registry.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 1]
```diff
@@ -397,6 +394,24 @@ Events:
...
+## 使用来自多个仓库的镜像
+
+一个 Pod 可以包含多个容器，每个容器的镜像可以来自不同的仓库。
+你可以在一个 Pod 中使用多个 `imagePullSecrets`，每个 `imagePullSecrets` 可以包含多个凭证。
+
+<!--
+The image pull will be attempted using each credential that matches the registry.
+If no credentials match the registry, the image pull will be attempted without authorization or using custom runtime specific configuration.
+-->
+将使用与仓库匹配的每个凭证尝试拉取镜像。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/pull-image-private-registry.md:412]:
```suggestion
kubelet 将使用与仓库匹配的每个凭证尝试拉取镜像。
```

File: content/zh-cn/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 2]
```diff
@@ -397,6 +394,24 @@ Events:
...
+如果没有凭证匹配仓库，则将尝试在没有授权的情况下拉取镜像，或者使用特定运行时的自定义配置。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/pull-image-private-registry.md:413]:
```suggestion
如果没有凭证匹配仓库，则 kubelet 将尝试在没有授权的情况下拉取镜像，或者使用特定运行时的自定义配置。
```

**Review by mengjiao-liu — COMMENTED** [review]
/lgtm

**Pull request merged**